guidelines/pride-affinity-proteomics.md

```diff
@@ -94,7 +93,7 @@ The following sections outline the specific requirements and file types for each
 
 ##### npx.csv/npx.parquet validations
 
-If your experiment is done on the latest version of Olink Explore (@Need a number here) then you must provide **.parquet file** which was exported from Olink Explore which includes the sample metadata, protein intensities, and NPX absolute quantification values. If the experiment is done on an older version of Olink Explore, then you can provide a **.csv file**
+If your experiment is done on the latest version of Olink Explore (version 3.0) then you must provide a **.parquet file** which was exported from Olink Explore which includes the sample metadata, protein intensities, and NPX absolute quantification values. If the experiment is done on an older version of Olink Explore, then you can provide a **.csv file**
```

Comment:
If your experiment is done on Olink Explore HT then you must provide a **.parquet file** which was exported from Olink Explore HT which includes the sample metadata, protein intensities, and NPX absolute quantification values. If the experiment is done on any other version of Olink Explore, then you can provide a **.csv file**